### PR TITLE
Sort benchmarks and handle exceptions generated by two new tests

### PR DESCRIPTION
**Pull request by zhangyiatmicrosoft**
Two fixes:
1. new tests (CustomLayer and MapIdle) were added but when running them by default <code>npm run benchmark</code> against v2.4.0 (main 7db0e81), null exceptions were thrown, causing the whole round to fail.
2. The benchmark tests are not executed in alphabetic order hence hard to keep track. Sorting them makes everything easier.

### CODE REVIEW

**Comment by zhangyiatmicrosoft** [comment]
main 7db0e81            main 383b88d  fixtests-dirty e603e0c
                 CustomLayer                                     0.0203 ms               0.0206 ms
            ExpressionCreate             0.3336 ms               0.3345 ms               0.3279 ms
          ExpressionEvaluate             0.0048 ms               0.0047 ms               0.0048 ms
          FeatureLevelRemove            66.0493 ms              65.6473 ms              66.3008 ms
                FilterCreate             0.4429 ms               0.4400 ms               0.4411 ms
              FilterEvaluate             0.3946 ms               0.4265 ms               0.4542 ms
              FunctionCreate             0.0237 ms               0.0235 ms               0.0238 ms
            FunctionEvaluate             0.0031 ms               0.0031 ms               0.0035 ms
               HillshadeLoad           350.0062 ms             346.7853 ms             345.3500 ms
             LayerBackground             1.5522 ms               1.5454 ms               1.5673 ms
                 LayerCircle             0.1531 ms               0.1610 ms               0.1583 ms
                   LayerFill             0.3100 ms               0.2954 ms               0.2969 ms
          LayerFillExtrusion             0.5356 ms               0.5439 ms               0.5407 ms
                LayerHeatmap             0.3825 ms               0.3844 ms               0.3822 ms
              LayerHillshade             0.5295 ms               0.5331 ms               0.5317 ms
                   LayerLine             0.1729 ms               0.1806 ms               0.1751 ms
                 LayerRaster             1.9934 ms               2.0331 ms               1.9738 ms
                 LayerSymbol             0.5868 ms               0.5884 ms               0.5901 ms
        LayerSymbolWithIcons             0.6478 ms               0.6455 ms               0.6523 ms⚠️
      LayerSymbolWithSortKey             0.3547 ms⚠️              0.3560 ms⚠️              0.3600 ms☠️
 LayerTextWithVariableAnchor            12.9015 ms              12.2326 ms              12.2154 ms
                      Layout           104.2772 ms             128.3389 ms             130.8346 ms
                        Load            33.3328 ms              33.6155 ms              33.3022 ms
                     MapIdle                                   669.2441 ms             683.3382 ms
                       Paint             9.9297 ms               9.9939 ms               9.9350 ms⚠️
                 PaintStates            68.6992 ms              68.2113 ms              69.2333 ms
                   Placement             9.1210 ms⚠️              9.5226 ms⚠️             14.6063 ms⚠️
         PropertyLevelRemove            67.8431 ms              68.5716 ms              67.4809 ms
                    QueryBox           120.9338 ms             118.7323 ms             119.0961 ms
                  QueryPoint            14.7969 ms⚠️             15.2553 ms⚠️             15.3717 ms⚠️
           SourceLevelRemove            68.2654 ms              67.5788 ms              67.4362 ms
            StyleLayerCreate             0.5882 ms               0.6535 ms⚠️              0.6665 ms
                SymbolLayout             7.3182 ms☠️              7.5309 ms               7.5910 ms⚠️
                    Validate             2.0964 ms               2.0242 ms⚠️              2.0467 ms
              WorkerTransfer            11.6124 ms⚠️             91.4820 ms              94.0836 ms

**Comment by zhangyiatmicrosoft** [comment]
main 7db0e81            main 383b88d  fixtests-dirty e603e0c
                 CustomLayer                                     0.0203 ms               0.0206 ms
            ExpressionCreate             0.3336 ms               0.3345 ms               0.3279 ms
          ExpressionEvaluate             0.0048 ms               0.0047 ms               0.0048 ms
          FeatureLevelRemove            66.0493 ms              65.6473 ms              66.3008 ms
                FilterCreate             0.4429 ms               0.4400 ms               0.4411 ms
              FilterEvaluate             0.3946 ms               0.4265 ms               0.4542 ms
              FunctionCreate             0.0237 ms               0.0235 ms               0.0238 ms
            FunctionEvaluate             0.0031 ms               0.0031 ms               0.0035 ms
               HillshadeLoad           350.0062 ms             346.7853 ms             345.3500 ms
             LayerBackground             1.5522 ms               1.5454 ms               1.5673 ms
                 LayerCircle             0.1531 ms               0.1610 ms               0.1583 ms
                   LayerFill             0.3100 ms               0.2954 ms               0.2969 ms
          LayerFillExtrusion             0.5356 ms               0.5439 ms               0.5407 ms
                LayerHeatmap             0.3825 ms               0.3844 ms               0.3822 ms
              LayerHillshade             0.5295 ms               0.5331 ms               0.5317 ms
                   LayerLine             0.1729 ms               0.1806 ms               0.1751 ms
                 LayerRaster             1.9934 ms               2.0331 ms               1.9738 ms
                 LayerSymbol             0.5868 ms               0.5884 ms               0.5901 ms
        LayerSymbolWithIcons             0.6478 ms               0.6455 ms               0.6523 ms⚠️
      LayerSymbolWithSortKey             0.3547 ms⚠️              0.3560 ms⚠️              0.3600 ms☠️

**Comment by zhangyiatmicrosoft** [comment]
main 7db0e81            main 383b88d  fixtests-dirty e603e0c
                 CustomLayer                                     0.0203 ms               0.0206 ms

      LayerSymbolWithSortKey             0.3547 ms⚠️              0.3560 ms⚠️              0.3600 ms☠️
 LayerTextWithVariableAnchor            12.9015 ms              12.2326 ms              12.2154 ms
                      Layout           104.2772 ms             128.3389 ms             130.8346 ms
                        Load            33.3328 ms              33.6155 ms              33.3022 ms
                     MapIdle                                   669.2441 ms             683.3382 ms
                       Paint             9.9297 ms               9.9939 ms               9.9350 ms⚠️
                 PaintStates            68.6992 ms              68.2113 ms              69.2333 ms
                   Placement             9.1210 ms⚠️              9.5226 ms⚠️             14.6063 ms⚠️
         PropertyLevelRemove            67.8431 ms              68.5716 ms              67.4809 ms
                    QueryBox           120.9338 ms             118.7323 ms             119.0961 ms
                  QueryPoint            14.7969 ms⚠️             15.2553 ms⚠️             15.3717 ms⚠️
           SourceLevelRemove            68.2654 ms              67.5788 ms              67.4362 ms
            StyleLayerCreate             0.5882 ms               0.6535 ms⚠️              0.6665 ms
                SymbolLayout             7.3182 ms☠️              7.5309 ms               7.5910 ms⚠️
                    Validate             2.0964 ms               2.0242 ms⚠️              2.0467 ms
              WorkerTransfer            11.6124 ms⚠️             91.4820 ms              94.0836 ms

File: test/bench/run-benchmarks.ts [hunk 1]
```diff
@@ -5,8 +5,17 @@ import minimist from 'minimist';
 
 const argv = minimist(process.argv.slice(2));
 
-const formatTime = (v) => `${v.toFixed(4)} ms`;
-const formatRegression = (v) => v.correlation < 0.9 ? '\u2620\uFE0F' : v.correlation < 0.99 ? '\u26A0\uFE0F' : ' ';
+const formatTime = (v) => {
+    return (typeof v === 'number') ? `${v.toFixed(4)} ms` : '';
```
Comment (HarelM) [inline test/bench/run-benchmarks.ts:9]: When is this not a number? We use typescript now, we should type the input parameter at least...

Comment (zhangyiatmicrosoft) [reply test/bench/run-benchmarks.ts]: Reverted this part. Thanks

**Review by HarelM — APPROVED** [review]

**Pull request merged**